src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: My Open Source Journey @Layer5
+subtitle: It changed my life completely!
+date: 2022-01-03 10:30:05 -0530
+author: Gaurav Chadha
+thumbnail: ./profile-banner.png
+description: "My journey as an Open Source contributor at Layer5. How I started from scratch, got involved in this community
+and gained many new experiences."
+category: Community
+tags:
+  - Open Source
+  - Experience
+  - Layer5
+  - Meshery
+published: true
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+import Profile from "./gaurav-chadha.png"
+
+<BlogWrapper>
+<img src={Profile} className="profileImg" align="right" alt="profile-img" />
+<p>
+  Hey👋, I am Gaurav Chadha, a student from New Delhi, India with lots of dreams, ambitions, and zeal to learn and grow more each and every day.
+</p>
+<h2 id="journey">How did it all begin?</h2>
+<p>
+My journey in Open source started with late-night surfing on LinkedIn,
+I saw a post about Layer5 organization and open-source which drove me to their website and slack channel, and then Boom my one click to join @Layer5's slack workspace kickstarted my journey in Open Source.
+I would say this was my best ever decision in life till now.
+</p>
+<h2 id="firs-pr">Getting started in the community</h2>
+<p>
+As soon as I joined Layer5's slack workspace, I landed on different slack channels workspace, At first I was a bit shy to introduce myself on the #introduction channel but then I saw many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community. 
+The best part here in Layer5 is each and every newcomer is warmly welcomed and 
+they also have a dedicated  Newcomers Call which helped them to get started with the project and to know more about how the community works.
+</p>
+<p>
+  I would highly recommend all the newcomers to engage as much as possible in the community, you will gradually start seeing a change in yourself.
+</p>
+
+<h3 id="first-pr">Way to my First Pull Request</h3>
+<p>
+I still remember at that time I was just a beginner with Git, HTML, CSS, and JavaScript and most of the issues on the repository were going over my head. 
+So I started with a very beginner-friendly issue about changing a background of a component and who would have thought this changing background issue will eventually change my life in open source. 
```

Comment:
@Chadha93 include links to the help wanted and the good first issue labels. Make sure the links that you include span all orgs. See any of the project repos' READMEs for an example.

Reply:
sounds good I'll add this